### PR TITLE
COMPLETE: Item Searcher Feature  + ItemSearchTest + ItemTest

### PR DESCRIPTION
**Pull request by kevinle623**
Due to the limitations of API calling, we have limited calls which is not practical for app use. From this, I have implemented webscraping using JSOUP to be able to item search by either a keyword or a direct url.

### CODE REVIEW

**Review by AlexChenl3 — APPROVED** [review]
everything looks good

File: src/main/java/ItemSearcher.java [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+//            for (Element item: productCounts){
+//                listCount.add(Integer.parseInt(item.text().replace(",","")));
+//            }
+
+//            System.out.println(listNames.size());
+//            System.out.println(listPrices.size());
+//            System.out.println(listUrls.size());
+//            System.out.println(listCount.size());
+//            System.out.println(listImgs.size());
+
```
Comment (Ljime) [inline src/main/java/ItemSearcher.java:195]: There are alot of commented code here, could you remove them?

File: src/test/java/DataBaseTest.java [hunk 2]
```diff
@@ -96,6 +96,7 @@ public void TestDataBaseWritesAndSavesListOfWishLists() {
 
         User user = new User("TestUser", "Test");
         DataBase.saveListOfWishlists(wishlists, user);
+        System.out.println(DataBase.getListOfWishlists("TestUser").getListOfWishlist().get(0));
```
Comment (Ljime) [inline src/test/java/DataBaseTest.java:99]: Any reason for this print statement being here?

**Review by Ljime — APPROVED** [review]
Good job on the new item searcher, only a couple of comments and print statements left code that should be cleaned up!

**Review by AlexChenl3 — DISMISSED** [review]

**Pull request closed without merging**